osx/data/script

```diff
@@ -14,7 +14,7 @@ source activate
 # setup the environment to not mess with the system
 export DYLD_FALLBACK_LIBRARY_PATH="${SCRIPT_PATH}/../lib:$DYLD_FALLBACK_LIBRARY_PATH"
 export LD_PRELOAD_PATH="${SCRIPT_PATH}/../lib"
-BUNDLE_ID=$(ls -1 $(pwd)/../../../MacOS)
+BUNDLE_ID=$(defaults read "${SCRIPT_PATH}/../Info)
```

Comment:
Ouch, seems you missed a closing `"` .

Reply:
Scratch that,  seems I was testing .app from a different source.

Reply:
BUNDLE_ID=$(osascript -e 'id of app "../../../../"')